src/battle_util.c

```diff
@@ -9484,6 +9484,10 @@ static u32 CalcFinalDmg(u32 dmg, u16 move, u8 battlerAtk, u8 battlerDef, u8 move
             finalModifier = uq4_12_multiply(finalModifier, UQ_4_12(1.5));
     }
 
+    // Collision Course, Electro Drift
+    if (gBattleMoves[move].effect == EFFECT_COLLISION_COURSE && typeEffectivenessModifier >= UQ_4_12(2.0))
+        finalModifier = uq4_12_multiply(finalModifier, UQ_4_12(1.33));
```

Comment:
Showdown says it actually boosts by 1.3333 instead of 1.33.
https://dex.pokemonshowdown.com/moves/collisioncourse

Reply:
Changed to 1.3333.